docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>"On the command line"</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>